CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### 0.4.4 (next)
 
+* [#36](https://github.com/dblock/iex-ruby-client/pull/36): Updates on largest-trades - [@rodolfobandeira](https://github.com/rodolfobandeira).
```

Comment:
Since we haven't shipped the largest trades stuff yet, no need to include this.

Reply:
danger-toc complaining about it.

Reply:
@dblock Rolled-back